packages/next/build/babel/plugins/next-page-config.ts

```diff
@@ -52,17 +59,29 @@ export default function nextPageConfig({
                 path: NodePath<BabelTypes.ExportNamedDeclaration>,
                 state: any
               ) {
-                if (
-                  state.bundleDropped ||
-                  !path.node.declaration ||
-                  !(path.node.declaration as any).declarations
-                )
+                if (state.bundleDropped || !path.node.declaration) {
                   return
-                const { declarations } = path.node.declaration as any
+                }
+                const { declarations, id } = path.node.declaration as any
                 const config: PageConfig = {}
 
+                // drop SSR Exports for client bundles
+                if (id && sprSSRExports.has(id.name)) {
+                  if (id.name === 'getStaticProps') {
```

Comment:
I don't like that this is a magic value... instead of a set above can we just create two constants?

`EXPORT_NAME_GET_STATIC_PROPS` and `EXPORT_NAME_GET_STATIC_PARAMS`?

Reply:
Updated